### ISSUE TITLE
Issue with importing asterisk community package

### ISSUE THREAD

**Issue by EricZ727** [amusecode/amuse#837]
Hello! I'm an undergrad student using AMUSE for my research, and I've been unable to get the [asterisk_movie_example](https://github.com/amusecode/amuse/blob/main/examples/applications/asterisk_movie_example.py) to work. I've been struggling to import the `amuse.community.asterisk.interface` package on line 8:

`(Amuse-env) (base) eric@DeStijl:/mnt/c/Users/ericz/OneDrive/Documents/Python/BRAVE$ /mnt/c/Users/ericz/OneDrive/Documents/Python/BRAVE/Amuse-env/bin/python /mnt/c/Users/ericz/OneDrive/Documents/Python/BRAVE/Tutorials/asterisk_movie_example.py
Traceback (most recent call last):
  File "/mnt/c/Users/ericz/OneDrive/Documents/Python/BRAVE/Tutorials/asterisk_movie_example.py", line 8, in <module>
    from amuse.community.asterisk.interface import Asterisk
ModuleNotFoundError: No module named 'amuse.community.asterisk'` 

I am using Windows 10, version 21H2, WSL1, Python 3.8.8, and NumPy 1.22.1. I think that my error lies in setting my CLASSPATH. Within the [readme file for asterisk](https://github.com/amusecode/amuse/blob/main/src/amuse/community/asterisk/src/README.md), it says to add the root directory to the classpath. I'm unsure of how to do that, I tried setting the entire asterisk folder as my classpath, but that didn't work. I am very new to AMUSE, and have no experience using Java and Python in conjunction with one another. Thanks for any help

**Comment by rieder** [comment]
Hi Eric,
Did you install AMUSE following the 'development' installation method, or did you install via `pip install amuse`?
The latter method does not currently include asterisk...

**Comment by EricZ727** [comment]
Hello!
I installed via the `pip install amuse`, so I'm trying to now install the "development" installation. 

I successfully installed IPython with the `pip install -e .` command documented [here](https://amuse.readthedocs.io/en/latest/install/howto-install-AMUSE.html#development-build). However, I'm don't understand what the instruction: 

> after this the codes need to be build
   `python setup.py develop_build`

Sorry for my ignorance in coding, Thanks so much.
